coremltools/converters/mil/mil/passes/defs/optimize_normalization.py

```diff
@@ -169,6 +170,7 @@ def _try_apply_transform(
         # check whether the pattern is instance_norm or layer_norm
         is_layernorm = False
         is_instancenorm = False
+        is_new_const = False
```

Comment:
what is `is_new_const`?
Can you add a comment explain it?

Reply:
This was a bool to signify whether gamma and beta needed to be materialized. I reorganized this function a little bit and removed `is_new_const` in the process.